### PR TITLE
EDEV-5 - Update tna-frontend to v0.1.23-prerelease

### PR DESCRIPTION
**Pull request by ahosgood**
https://national-archives.atlassian.net/browse/EDEV-5

Updated `tna-frontend` to [`v0.1.23-prerelease`](https://github.com/nationalarchives/tna-frontend/compare/v0.1.20-prerelease...v0.1.23-prerelease) from `v0.1.20-prerelease`

- https://national-archives.atlassian.net/browse/UN-569
- https://national-archives.atlassian.net/browse/UN-615
- Removed card supertitles in "Explore by topic" and "Explore by time period" pages
- Changed medium and small headings back to Open Sans

Review site: https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/

### CODE REVIEW

File: .github/workflows/platformsh-cd-develop.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
       # - main
       - develop
+      - feature/frontend-0-1-22 # TEMP
```
Comment (ahosgood) [inline .github/workflows/platformsh-cd-develop.yml:8]: TODO: Remove before merge

**Comment by JohnHeeryTNA** [comment]
Hi @ahosgood the following are some visual changes I've noticed and wanted to flag. They may well be correct as design but I just wanted to check..

**Comment by JohnHeeryTNA** [comment]
https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/

Screenshot 2023-10-27 at 10 12 15

We have a bottom border on the `featured-article__heading` is that correct?

The button has `text-align:center`, is this consistent with other buttons?

**Comment by JohnHeeryTNA** [comment]
https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/stories/Policing-migration/

Screenshot 2023-10-27 at 11 15 06

The intro text seems to be 2/3rds width `tna-column--width-2-3` is that correct?

**Comment by JohnHeeryTNA** [comment]
https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/stories/the-battle-of-the-somme/
Screenshot 2023-10-27 at 10 27 06

The `tna-blockquote__quote` sections have `font-weight:700` is this per design?

**Comment by JohnHeeryTNA** [comment]
https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/search/

On search (which isn't live yet so this may not need attention) only one menu item is appearing on mobile/desktop
Screenshot 2023-10-27 at 10 16 10

Screenshot 2023-10-27 at 10 17 29

**Comment by ahosgood** [comment]
> https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/
> 
> Screenshot 2023-10-27 at 10 12 15
> We have a bottom border on the `featured-article__heading` is that correct?
> 
> The button has `text-align:center`, is this consistent with other buttons?

That underline has always been there. It may be removed in the future but it's in the designs.

image

The buttons are always centre-aligned now, however I feel this button has far too much text in it. We may need to chat with content designers to come up with a sensible limit for this.

**Comment by ahosgood** [comment]
> https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/stories/Policing-migration/
> 
> Screenshot 2023-10-27 at 11 15 06
> The intro text seems to be 2/3rds width `tna-column--width-2-3` is that correct?

It's as close as I could get given the design was done before we had a proper grid system to adhere to. 😅

**Comment by ahosgood** [comment]
> https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/explore-the-collection/stories/the-battle-of-the-somme/ Screenshot 2023-10-27 at 10 27 06
> 
> The `tna-blockquote__quote` sections have `font-weight:700` is this per design?

I couldn't find a blockquote style in the designs but this hasn't changed from what's already on `develop`.

**Comment by ahosgood** [comment]
> https://feature-frontend-0-1-22-k3akvoq-rasrzs7pi6sd4.uk-1.platformsh.site/search/
> 
> On search (which isn't live yet so this may not need attention) only one menu item is appearing on mobile/desktop Screenshot 2023-10-27 at 10 16 10
> 
> Screenshot 2023-10-27 at 10 17 29

That's intentional at the moment and the same as `develop` 😊

**Review by JohnHeeryTNA — APPROVED** [review]
approving changes.

**Pull request merged**